### PR TITLE
Sidebar

### PR DESCRIPTION
**Pull request by jroycodes**

### CODE REVIEW

File: scss/styles.css [hunk 1]
```diff
@@ -35,6 +35,80 @@
...
+  height: 100%;
+  width: 0;
+  position: fixed;
+  top: 0;
+  right: 0;
+  overflow: hidden;
+  transition: 0.5s;
+  padding-top: 3.75rem;
+  z-index: 1;
+  background-color: #FFFFFF;
```
Comment (codingknite) [inline scss/styles.css]: use variable -- `$white`

File: scss/styles.css [hunk 2]
```diff
@@ -35,6 +35,80 @@
...
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  gap: 1rem;
+  cursor: pointer;
+  text-align: left;
+}
+.baritem a {
+  text-decoration: none;
+  color: #000000;
```
Comment (codingknite) [inline scss/styles.css:63]: use variable, maintain consistency

File: scss/styles.css [hunk 3]
```diff
@@ -35,6 +35,80 @@
...
+}
+
+a:first-child {
+  color: #fc7f3a;
```
Comment (codingknite) [inline scss/styles.css:67]: same here

File: scss/styles.css [hunk 4]
```diff
@@ -35,6 +35,80 @@
...
+.openbtn:hover {
+  color: #fc7f3a;
+}
+
+.openbtn {
+  display: block;
+  font-size: 1.7rem;
+  cursor: pointer;
+  float: right;
+  background-color: #FFFFFF;
```
Comment (codingknite) [inline scss/styles.css:100]: and here

File: index.html [hunk 5]
```diff
@@ -13,20 +13,30 @@
         <img src="Assets/Mobile/Logo.svg" alt="Healthcare logo" />
       </div>
 
+      <!-- Sidebar -->
+      <div id="main">
+        <button class="openbtn" id="button" onclick="openNav()">☰</button>
```
Comment (codingknite) [inline index.html]: use image or icon

File: index.html [hunk 6]
```diff
@@ -13,20 +13,30 @@
...
+          <a href="about.html">About</a>
+          <a href="about.html">Services</a>
+          <a href="blog.html">Blog</a>
+          <a href="blog.html">Pricing</a>
+        </div>
+      </div>
+
       <!-- DESKTOP MENU -->
-      <div class="desktop_menu" style="display: none;">
+      <div class="desktop_menu" style="display: none">
```
Comment (codingknite) [inline index.html]: avoid inline styles, use a class

File: script.js [hunk 7]
```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+// Opens sidebar when clicked
+function openNav() {
+  document.getElementById("mySidebar").style.width = "250px";
```
Comment (codingknite) [inline script.js]: use rem instead of px whenever possible

**Review by codingknite — COMMENTED** [review]
work on the stuff i've mentioned, the rest looks good to me

File: scss/styles.css (hunk 4 above)
Comment (jroycodes) [reply scss/styles.css:100]: Screenshot 2023-10-17 075714

Styles in the Css file defaulted back after Scss compiled

**Comment by jroycodes** [comment]
> work on the stuff i've mentioned, the rest looks good to me

Done!! Appreciate it.

**Pull request merged**